packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json

```diff
@@ -9,6 +9,20 @@
   "telephony_line_softphone_download_links_title": "Liens de téléchargements",
   "telephony_line_softphone_configure_title": "Configurer la ligne",
   "telephony_line_softphone_configure_description": "Vous avez la possibilité de configurer jusqu'à trois appareils au total. Vous pouvez configurer un seul appareil à la fois.",
+  "telephony_line_softphone_add_device": "Ajouter un appareil",
+  "telephony_line_softphone_add_my_device": "Ajouter mon appareil",
+  "telephony_line_softphone_record_link": "Lien d'enregistrement",
+  "telephony_line_softphone_add_device_description_bold": "Pour configurer votre ligne, veuillez nommer votre appareil et générer un lien.",
+  "telephony_line_softphone_add_device_description": "Vous pouvez ensuite scanner le QR Code ou utiliser le lien pour enregistrer votre appareil.",
+  "telephony_line_softphone_add_device_label": "Nom de mon appareil (obligatoire)",
+  "telephony_line_softphone_generate_link": "Générer un lien",
+  "telephony_line_softphone_device_name_placeholder": "Appareil 1",
```

Comment:
```suggestion
  "telephony_line_softphone_device_name_placeholder": "Appareil {{ deviceNumber }}",
```

Reply:
even if deviceNumer is and will always be 1 ? (it's a placeholder)